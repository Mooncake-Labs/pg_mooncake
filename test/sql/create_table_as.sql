LOAD 'pg_mooncake';

CREATE TABLE s (a int);
INSERT INTO s VALUES (1), (2), (3);

CREATE TABLE t AS SELECT * FROM s;
SELECT * FROM t;
DROP TABLE t;

CREATE TABLE t USING columnstore AS SELECT * FROM s;

DROP TABLE s;
