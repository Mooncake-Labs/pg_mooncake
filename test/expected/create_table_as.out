LOAD 'pg_mooncake';
CREATE TABLE s (a int);
INSERT INTO s VALUES (1), (2), (3);
CREATE TABLE t AS SELECT * FROM s;
SELECT * FROM t;
 a 
---
 1
 2
 3
(3 rows)

DROP TABLE t;
CREATE TABLE t USING columnstore AS SELECT * FROM s;
ERROR:  CREATE TABLE AS USING columnstore is not supported
DROP TABLE s;
